src/oscar/models/fields/__init__.py

```diff
@@ -73,14 +74,31 @@ class UppercaseCharField(CharField):
     Defined with the with_metaclass helper so that to_python is called
     https://docs.djangoproject.com/en/1.6/howto/custom-model-fields/#the-subfieldbase-metaclass  # NOQA
     """
-
```

Comment:
Please avoid unnecessary white space changes, thanks!

Reply:
thank you your comment. I fix right now